VmAgent.Core/Interfaces/SystemOperations.cs

```diff
@@ -68,6 +69,11 @@ public void DeleteDirectoryIfExists(string pathToDirectory)
             }
         }
 
+        public bool IsDirectoryEmpty(string pathToDirectory)
+        {
+            return Directory.EnumerateFileSystemEntries(pathToDirectory).Any();
```

Comment:
Any() returns trye if there are any. Should we rename this method or return the opposite?

Reply:
Yep good catch